lms/templates/ccx/coach_dashboard.html

```diff
@@ -155,4 +170,22 @@
       $('#ccx_std_list_messages')[0].focus();
     }
   });
+  function validateForm(form) {
+    var newCCXName = $(form).find('#ccx_name').val();
+    var $errorMessage =  $('#ccx-create-message');
+    var hasCcxConnector = ${"true" if course and hasattr(course, 'ccx_connector') and course.ccx_connector else "false"};
+
+    if ((!newCCXName || newCCXName === "") && !hasCcxConnector) {
```

Comment:
`!newCCXName` will always be true if `newCCXName === ""` is true. Can you remove `newCCXName === ""`?